.travis.yml

```diff
@@ -13,4 +14,4 @@ rvm:
   - truffleruby
 
 install: ( cd api && bundle install --jobs=3 --retry=3 )
-script: ( cd $TEST_DIR && rake )
+script: ( if [ $TEST_DIR=='bridges/opentracing' ]; then gem install opentracing -v 0.5.0; fi && cd $TEST_DIR && rake )
```

Comment:
Why is this necessary? Is this because we can't run `bundle install` in the `bridges/opentracing` directory?

Reply:
💯